README.md

```diff
@@ -1,68 +1,29 @@
-
-## Contents
-* app.R
-    * Launch the 19andMe app (https://19andme.covid19.mathematica.org)
-* data/
-    * Store the input data
-* doc/
-    * Calculate input from the literature
-* src/
-    * R Scripts for different modules of the app
-    * Google Analytics plug-in
-    * CSS
-* www/
-    * Mathematica logo
-## Authors
-* **Cindy Hu** - *back-end lead*
-* **George Luo** - *front-end lead*
-* **Emma Pendl-Robinson, Erin Lipman** - *back-end developer*
-
-## Reviewers and Support
-* **Margaret Luo** - *Docker guru*
-* **Arpan Bhattacharya, George Gallo** - *AWS architects*
-* **Kelsey Skvoretz** - *QA reviewer*
-* **Fei Xing** - *QA reviewer*
-* **Matt Salganik** - *Expert reviewer*
-
-## Acknowledgements
-We appreciate additional support towards this project from Alex Bohl, Dave Peters, and Matt Gillingham.
-
-## Contact info
-For more information, please contact Cindy Hu at CHu@mathematica-mpr.com and George Luo at covid.risk.score@gmail.com.
-
-# DockerShinyApp
-
-## Installing
-This project is built to use Docker and docker-compose to make development easy across all machines and remove host machine configuration as a potential issue.  Use `docker-compose` to get started quickly.  You will need to install docker and docker-compose. 
-
-* [Docker](https://docs.docker.com/install/)
-* [Docker Compose](https://docs.docker.com/compose/install/)
-
-Use docker-compose to build the image:
-```
-docker-compose build
-```
-
-Remember that docker images are immutable once built.  Only changes to files in `/home/rstudio` will persist after restarts. 
-
-## Development
-To create an RStudio environment preloaded with all dependencies
-1. Create a `.env` file that defines the desired RStudio password, e.g.
-```
-PASSWORD=mystrongpassword
-```
-There is an example [sample.env](sample.env) you can also use.  Copy it and rename it to `.env`. 
-
-2. Generate username and API key for USPS ZIP CODE CROSSWALK [here](https://www.huduser.gov/portal/dataset/uspszip-api.html). We will use this to look up the FIPS county code for the zip code submitted by the user. Add API key to doc/HUD_API_KEY.txt
-
-3. Start the environment
-`docker-compose up`
-
-4. Visit `http://localhost:8787` and start hacking.
-
-5. Keep shiny app code in `app.R`.  Launch it for development with `shiny::runApp('app.R')`.
-
-## Deployment
-Utilize the attached Dockerfile for a simple but efficient deployment setup.
+# 19andMe
+An interactive dashboard for users to calculate their individualized risk score of contracting COVID-19 and related adverse health outcomes.
+
+## Contents
+
+* app/
+    * R Scripts for different modules of the app
+    * Google Analytics plug-in
+    * CSS
+    * qwww/
+        * Mathematica logo
+
+## Authors
+* **Cindy Hu** - *back-end lead*
+* **George Luo** - *front-end lead*
+* **Emma Pendl-Robinson, Erin Lipman** - *back-end developer*
+
+## Reviewers and Support
+* **Margaret Luo** - *Docker guru*
```

Comment:
I personally don't really like the use of the word guru in this context because it feels like appropriation from Buddhist and Hindu religions. Could we use the word "expert" or "implementer" instead?

Reply:
Thanks Margaret for your comment, I agree. 
I think this change should be made in the readme pr (https://github.com/mathematica-mpr/covid_risk_score/pull/51)